api/src/main/java/org/openmrs/module/Module.java

```diff
@@ -718,6 +718,10 @@ public boolean isStarted() {
 		return ModuleFactory.isModuleStarted(this);
 	}
 	
+	/**
+	 * @param e string to set as startup error message
+	 * @should throw exception when message is null
```

Comment:
@ern2 my recommendation is that you spend your energy while prioritizing on real functionality tests instead of simple null argument validation tests. :)